server/controller.py

```diff
@@ -7,25 +7,22 @@
 
 import asyncio
 import json
-import os
 import ssl
-import random
 import time
 import traceback
-import warnings
 from bisect import bisect_left
-from collections import defaultdict
-from concurrent.futures import ThreadPoolExecutor
 from functools import partial
 
+import os
 import pylru
+from collections import defaultdict
+from concurrent.futures import ThreadPoolExecutor
 
-from lib.jsonrpc import JSONRPC, JSONSessionBase, RPCError
+import lib.util as util
 from lib.hash import double_sha256, hash_to_str, hex_str_to_hash
+from lib.jsonrpc import JSONSessionBase, RPCError
 from lib.peer import Peer
-import lib.util as util
-from server.block_processor import BlockProcessor
-from server.daemon import Daemon, DaemonError
+from server.daemon import DaemonError
 from server.mempool import MemPool
 from server.peers import PeerManager
 from server.session import LocalRPC, ElectrumX
```

Comment:
John - I like to keep the imports split into 3 as per PEP8 I think - Python native imports first, 3rd party libs second, and then the app itself 3rd.  The change looks good apart from this, though I am unable to test it at the moment.
Can you restore the import order please and let me know if you manage to give it a test.  Thanks.